e2e/specs/menu_bar/view_menu.test.js

```diff
@@ -110,36 +122,84 @@ describe('menu/view', function desc() {
         zoomLevel.should.be.equal(1);
     });
 
-    it('MM-T818 Zoom in from the menu bar', async () => {
-        const mainWindow = this.app.windows().find((window) => window.url().includes('index'));
-        const browserWindow = await this.app.browserWindow(mainWindow);
-        const loadingScreen = this.app.windows().find((window) => window.url().includes('loadingScreen'));
-        await loadingScreen.waitForSelector('.LoadingScreen', {state: 'hidden'});
-        const firstServer = this.serverMap[`${config.teams[0].name}___TAB_MESSAGING`].win;
-        const firstServerId = this.serverMap[`${config.teams[0].name}___TAB_MESSAGING`].webContentsId;
-        await env.loginToMattermost(firstServer);
-        await firstServer.waitForSelector('#searchBox');
+    describe('MM-T818 Zoom in from the menu bar', () => {
+        it('MM-T818 Zoom in when CmdOrCtrl+Plus is pressed', async () => {
+            const mainWindow = this.app.windows().find((window) => window.url().includes('index'));
+            const browserWindow = await this.app.browserWindow(mainWindow);
+            const loadingScreen = this.app.windows().find((window) => window.url().includes('loadingScreen'));
+            await loadingScreen.waitForSelector('.LoadingScreen', {state: 'hidden'});
+            const firstServer = this.serverMap[`${config.teams[0].name}___TAB_MESSAGING`].win;
+            const firstServerId = this.serverMap[`${config.teams[0].name}___TAB_MESSAGING`].webContentsId;
+            await env.loginToMattermost(firstServer);
+            await firstServer.waitForSelector('#searchBox');
 
-        robot.keyTap('=', [env.cmdOrCtrl]);
-        await asyncSleep(1000);
-        const zoomLevel = await browserWindow.evaluate((window, id) => window.getBrowserViews().find((view) => view.webContents.id === id).webContents.getZoomFactor(), firstServerId);
-        zoomLevel.should.be.greaterThan(1);
+            robot.keyTap('=', [env.cmdOrCtrl]);
+            await asyncSleep(1000);
+            const zoomLevel = await browserWindow.evaluate((window, id) => window.getBrowserViews().find((view) => view.webContents.id === id).webContents.getZoomFactor(), firstServerId);
+            zoomLevel.should.be.greaterThan(1);
+        });
+
+        it('MM-44931 Zoom in when CmdOrCtrl+Shift+Plus is pressed', async () => {
```

Comment:
All looks good, I just wanted to name the test cases for the test cases we have stores in JIRA.

```suggestion
        it('MM-T818_1 Zoom in when CmdOrCtrl+Shift+Plus is pressed', async () => {
```